cdc/log.cc

```diff
@@ -391,16 +387,18 @@ cdc::options::options(const std::map<sstring, sstring>& map) {
 }
 
 std::map<sstring, sstring> cdc::options::to_map() const {
-    if (!_enabled) {
-        return {};
-    }
-    return {
-        { "enabled", _enabled ? "true" : "false" },
+    std::map<sstring, sstring> ret = {
         { "preimage", to_string(_preimage) },
         { "postimage", _postimage ? "true" : "false" },
         { "delta", to_string(_delta_mode) },
         { "ttl", std::to_string(_ttl) },
     };
+
+    if (_enabled) {
```

Comment:
we should probably expect _enabled to be always set here and if not call on_internal_error.

Reply:
After futher thinking I restored the original behavior - it looks clean and does not pose a risk of messing up any integration tests.

Reply:
Anyway, it was just boy-scouting: allowing to print invalid `cdc_options` for the purpose of e.g. debugging. Currently it would be unneeded anyways.

Reply:
After even further thinking I changed the semantics to something in-between. Only the unset `cdc_options` will be printed as `{}`.
CDC dtests do pass.